dtfs-central-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -5,9 +5,14 @@ const bankIdValidation = param('bankId').isString().matches(/^\d+$/).withMessage
 
 exports.bankIdValidation = [bankIdValidation];
 
-const mongoIdValidation = param('_id').isMongoId().withMessage("Invalid MongoDB '_id' path param provided");
+/**
+ * Validator for a path parameter which is a mongo id
+ * @param {string} paramName
+ * @returns {import('express-validator').ValidationChain}
+ */
+const mongoIdValidation = (paramName) => param(paramName).isMongoId().withMessage("Invalid MongoDB '_id' path param provided");
```

Comment:
We sometimes have multiple mongoIds in one URL. Do we want to add the param name to the message

Reply:
Ah yes, that was my mistake. I'll update that